.oscal/ssp.oscal.yaml

```diff
@@ -0,0 +1,289 @@
+        title: Security Engineer
+      - id: prepared-by
+        title: Document Preparer
+      - id: prepared-for
+        title: Document Recipient
+    
+    parties:
+      - uuid: 11111111-0000-4000-9000-100000000001
+        type: organization
+        name: "Metaschema Framework Organization"
+        short-name: "MFO"
+      - uuid: 11111111-0000-4000-9000-100000000002
+        type: person
+        name: "David Waltermire"
+        props:
+          - name: job-title
+            value: Software Architect
+      - uuid: 22222222-0000-4000-9000-100000000002
+        type: person
+        name: "Paul Wand"
+        props:
+          - name: job-title
+            value: Senior Software Architect
+      - uuid: 88888888-0000-4000-9000-100000000008
+        type: person
+        name: "John Doe"
+        props:
+          - name: job-title
+            value: Document Preparer
+      - uuid: 99999999-0000-4000-9000-100000000009
+        type: organization
+        name: "FedRAMP PMO"
+        addresses:
+          - addr-lines:
+              - "1234 FedRAMP Way"
+            city: "Washington"
+            state: "DC"
+            postal-code: "20001"
+
+    responsible-parties:
+      - role-id: system-owner
+        party-uuids:
+          - 11111111-0000-4000-9000-100000000002
+      - role-id: information-security-officer
+        party-uuids:
+          - 22222222-0000-4000-9000-100000000002
+      - role-id: prepared-by
+        party-uuids:
+          - 88888888-0000-4000-9000-100000000008
+      - role-id: prepared-for
+        party-uuids:
+          - 99999999-0000-4000-9000-100000000009
+
+    props:
+      - name: marking
+        value: "SENSITIVE BUT UNCLASSIFIED"
+
+  import-profile:
+    href: "profile.oscal.yaml"
+
+  system-characteristics:
+    system-ids:
+      - id: "ESC-DEV-001"
+        identifier-type: https://fedramp.gov
+    system-name: "Enterprise Software Development Environment"
+    system-name-short: "ESDE"
+    description: >-
+      Primary software development environment for Example Software Company,
+      including development, testing, and build infrastructure.
+    security-sensitivity-level: fips-199-moderate
+    system-information:
+      information-types:
+        - title: "Software Development Information"
+          uuid: 370026f8-582b-4c02-9331-a5c84afdbfd8
+          description: "Source code, build configurations, and development artifacts"
+          confidentiality-impact:
+            base: fips-199-moderate
+            selected: fips-199-moderate
+            adjustment-justification: "Default FIPS-199 moderate impact"
+          integrity-impact:
+            base: fips-199-moderate
+            selected: fips-199-moderate
+            adjustment-justification: "Default FIPS-199 moderate impact"
+          availability-impact:
+            base: fips-199-moderate
+            selected: fips-199-moderate
+            adjustment-justification: "Default FIPS-199 moderate impact"
```

Comment:
Can you please select an explicit information categorization, C.3.5.1 for system development information, and adjust the bases accordingly?

```suggestion
      information-types:
        - title: "Software Development Information"
          uuid: 370026f8-582b-4c02-9331-a5c84afdbfd8
          description: "Source code, build configurations, and development artifacts"
          categorization:
            system: https://doi.org/10.6028/NIST.SP.800-60v2r1
            information-type-id: C.3.5.1
          confidentiality-impact:
            base: fips-199-low
            selected: fips-199-low
          integrity-impact:
            base: fips-199-moderate
            selected: fips-199-low
            adjustment-justification: >-
              This library is provided as a public domain application built with free, open-source, and public domain libraries. Vendors using this application must review and confirm the integrity of the codebase and its changes over time and use the Merkle-tree data structure with integrity checking mechanisms as provided by the `git` utility, protocol, and GitHub code sharing service. Integrity checking is free and ubiqutous, and the impact of integrity violation to this code base is low, not moderate.
          availability-impact:
            base: fips-199-moderate
            selected: fips-199-low
```

Reply:
added